src/System.Runtime.Extensions/tests/System/IO/Path.IsPathFullyQualified.cs

```diff
@@ -12,8 +12,9 @@ public static class GetFullyQualifiedPathTests
         public static void IsPathFullyQualified_NullThrows()
```

Comment:
Perhaps change the name of the test? I don't recall if after your update in the coreclr PR this will still throw though.

Reply:
It will throw for string overload but not for span overload. Should I make different test for span or just change it to Assert.equal() for span overload